articles/ros2_dds_security_pkcs11_support.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: ROS2 DDS security PKCS#11 URI support
+abstract:
+  The [DDS-Security specification][dds_security] defines the use of Hardware Security Modules (HSM) and PKCS#11 URIs as an alternative to private keys and certificates stored in the file system.
+  Current implementation only supports these tokens to be directly stored in the file system as `.pem` files.
+  This is a design proposal to support PKCS#11 URIs.
+author: '[Iker Luengo](https://github.com/IkerLuengo)'
+published: false
+---
+
+- This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Scope
```

Comment:
I take this into more like `Motivation` section? especially the end of this section would look like problem statement.